scripts/configure-rdsh.ps1

```diff
@@ -0,0 +1,24 @@
+[CmdLetBinding()]
+Param(
+    $ServerFQDN,
+    $DomainNetBiosName="DICELAB",
+    $GroupName="Dicelab Remote Access Users"
+    )
+
+#Based on:
+# * hhttps://s3.amazonaws.com/app-chemistry/scripts/configure-rdsh.ps1
+
+if (-not $ServerFQDN) {
+    $name = invoke-restmethod -uri http://169.254.169.254/latest/meta-data/public-hostname
+    if (-not $name) {
+        $name = [System.Net.DNS]::GetHostByName('').HostName
+    }
+    $ServerFQDN = $name 
+}
+
+$null = Install-WindowsFeature RDS-RD-Server
+$null = Import-Module remotedesktopservices
+
+([ADSI]"WinNT://$env:COMPUTERNAME/Remote Desktop Users,group").Add("WinNT://$DomainNetBiosName/$GroupName,group")
```

Comment:
wrap this in an if statement, and might as well make it stateful while we're at it. i.e.:

```powershell
if ($DomainNetBiosName -and $GroupName) {
    $group = [ADSI]"WinNT://$env:COMPUTERNAME/Remote Desktop Users,group"
    $groupmembers = @(@($group.Invoke("Members")) | `
        foreach {$_.GetType().InvokeMember("Name", 'GetProperty', $null, $_, $null)})
    if ($groupmembers -notcontains $GroupName) {
        group.Add("WinNT://$DomainNetBiosName/$GroupName,group")
    }
}
```